labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,20 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi" +
+      "rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd" +
+      "de kr kd eoya kw aej tysr re ujdr lkgc jv" +
+      "aoz"
+    val output = "our language is impossible to understand" +
+     "there are twenty six factorial possibilities" +
+     "so it is okay if you want to just give up" +
+     "yeq"
+     
+    val mapping = input.zip(output).toMap
+    
+    lines map {
+      _.map(mapping(_))
+    }
```

Comment:
`lines map` is calling map on the sequence of strings, and then `_.map` is calling map on the string in the sequence.

It probably would be more clear as
```
lines map {
    _ map {
        mapping (_)
    }
}
```